ts/features/payments/onboarding/screens/PaymentsOnboardingFeedbackScreen.tsx

```diff
@@ -61,6 +73,17 @@ const PaymentsOnboardingFeedbackScreen = () => {
     Object.values(WalletOnboardingOutcomeEnum).indexOf(outcome)
   ] as keyof typeof WalletOnboardingOutcomeEnum;
 
+  useOnFirstRender(() => {
+    const onboardedPaymentMethodName = availablePaymentMethods?.find(
+      paymentMethod => paymentMethod.id === selectedPaymentMethodId
+    )?.name;
+    if (outcome === WalletOnboardingOutcomeEnum.SUCCESS) {
+      analytics.trackSuccessOnboardingPaymentMethod({
+        payment_method_selected: onboardedPaymentMethodName
```

Comment:
You can pass this as shorthand

Reply:
Are you referring to the onboardedPaymentMethodName? This constant is valuable in another iteration where tracking failure outcomes is necessary, ensuring that each outcome includes the payment_method_selected. I created it to have a readily available constant containing this information, making it reusable across other analytics methods that will be implemented in the next iteration as highlighted in this [jira comment](https://pagopa.atlassian.net/browse/IOBP-985?focusedCommentId=187609)

Reply:
I mean you can rename it as `payment_method_selected` and then below pass it as shorthand. What do you think?

Reply:
It makes sense! Addressed into 42ce8deac8fe84b6d478045469f04aaa32a5f14a